docs/config-olap-databases.md

```diff
@@ -4,26 +4,38 @@ title: OLAP Database Configuration
 sidebar_label: OLAP Database
 ---
 
-## Section
+## Introduction
 
-`olap_database:` The OLAP (online analytical processing) database section configures the OLAP database that BuildBuddy uses to enable the Trends page. **Optional**
+To speed up the analysis of historical build event data, BuildBuddy can be configured to use ClickHouse as an OLAP database, in addition to the primary SQL database required for core functionality.
```

Comment:
Would be nice to list a few features that depend on clickhouse.
- Historical build events (Trends page)
- Test Grid
- Tags
- Audit Logs

Reply:
I added tag filtering and audit logging to the list of features depending on OLAP DB.